File: 프로그래머스/0/181920. 카운트 업/카운트 업.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(int start_num, int end_num) {
    // return 값은 malloc 등 동적 할당을 사용해주세요. 할당 길이는 상황에 맞게 변경해주세요.
    int* answer = (int*)malloc((end_num - start_num + 1) * sizeof(int));
    int n=0;
    for(int i=start_num; i<=end_num;i++){
        answer[n]=i;
        n++;
    }
    return answer;
}

File: 프로그래머스/0/181920. 카운트 업/카운트 업.cs
using System;

public class Solution {
    public int[] solution(int start_num, int end_num) {
        int[] answer = new int[end_num-start_num+1];
        int n=0;
        for(int i=start_num; i<=end_num; i++){
            answer[n]= i;
            n++;
        }
        return answer;
    }
}